madanalysis/IOinterface/library_writer.py

```diff
@@ -263,6 +263,7 @@ def WriteMakefileForInterfaces(self,package):
             options.has_root        = True
             options.has_root_inc    = True
             options.has_root_lib    = True
+            options.ma5_fastjet_mode = self.main.archi_info.has_root and self.main.archi_info.has_fastjet
```

Comment:
Why are we compiling it with fastjet flag when there is root? This is only used when there is delphes, root can be used just for plotting even in hadron or parton modes

Reply:
This is needed to have delphes compiled with the fastjet flag when fastjet is present. Otherwise the library is not compiled as needed.